Automattic/wp-calypso
# Update sites per plugin view for bulk management

Fixes https://github.com/Automattic/dotcom-forge/issues/9923

## Proposed Changes

* Adds a [new route](https://github.com/Automattic/wp-calypso/pull/97697/files#diff-7edd3c450e9fa353a0acd99fdf2b07d0c902ef5c88f419a3f9ddd3df242822f8) that loads the new plugins dashboard
* Adds a [condition](https://github.com/Automattic/wp-calypso/pull/97697/files#diff-33316ee02ed424c6862ab81c9792fd225a851305558dca401c9029a2dc2f94fb) so that Global Sidebar collapses as expected  
* Created a new standalone functional component [Plugins Dashboard](https://github.com/Automattic/wp-calypso/pull/97697/files#diff-8d1828ee17b44e66577d242e108e3f5216f9fbc9beb675ba587bab287bd7d9c4) that shows the plugins list and a flyout window when a plugin is clicked


## Known Issues

- [x] Styling per WHjPtm5mPBnJyiBYVHHnDT-fi-794_72587 and some mobile styling.
- [x] Bulk selecting plugins and activating / deactivating only works the second time attempted.
- [ ] The components used are from the `calypso/layout/multi-sites-dashboard` and use a lot of related classes. Need to come up with a better plan.

## Testing Instructions



* Visit `plugins/manage/sites` in this branch and `plugins/manage` in production
* Check that the dataview is rendered the same way
* Try bulk actions and make sure there are no regressions
* Click a plugin and check if the sites loaded are the same and if related actions work

## Pre-merge Checklist
